src/components/portfolio/DetailsMainDividends.tsx

```diff
@@ -105,36 +105,39 @@ const DetailsMainDividends: React.FC<DetailsMainDividendsProps> = ({
     },
   ];
 
-  return (
-    <div className={classes.dividendsWrapper}>
-      <div className={classes.chartContainer}>
-        {/* left side with graph */}
-        <DividendLineChart
-          series={series}
-          height={450}
-          textColor={theme.palette.primary.contrastText}
-        />
-      </div>
-      <div className={classes.infoContainer}>
-        {/* right side with info */}
-        <InfoBlock title={t('portfolio.details.divYield')}>
-          {portfolio.keyFigures[portfolio.keyFigures.length - 1].div}
-        </InfoBlock>
-        <InfoBlock title={t('portfolio.details.divPayout')}>
-          <RatioDonut
-            ratio={
-              portfolio.keyFigures[portfolio.keyFigures.length - 1]
-                .dividendPayoutRatio
-            }
+  if (portfolio.keyFigures.length !== 0) {
+    return (
+      <div className={classes.dividendsWrapper}>
+        <div className={classes.chartContainer}>
+          {/* left side with graph */}
+          <DividendLineChart
+            series={series}
+            height={450}
             textColor={theme.palette.primary.contrastText}
           />
-        </InfoBlock>
-        <InfoBlock title={t('portfolio.details.nextDate')}>
-          {portfolio.nextDividend.toLocaleDateString()}
-        </InfoBlock>
+        </div>
+        <div className={classes.infoContainer}>
+          {/* right side with info */}
+          <InfoBlock title={t('portfolio.details.divYield')}>
+            {portfolio.keyFigures[portfolio.keyFigures.length - 1].div}
+          </InfoBlock>
+          <InfoBlock title={t('portfolio.details.divPayout')}>
+            <RatioDonut
+              ratio={
+                portfolio.keyFigures[portfolio.keyFigures.length - 1]
+                  .dividendPayoutRatio
+              }
+              textColor={theme.palette.primary.contrastText}
+            />
+          </InfoBlock>
+          <InfoBlock title={t('portfolio.details.nextDate')}>
+            {portfolio.nextDividend.toLocaleDateString()}
+          </InfoBlock>
+        </div>
       </div>
-    </div>
-  );
+    );
+  }
+  return <div className={classes.infoBody}>No information available yet</div>;
```

Comment:
Translation key needed. Also, shouldn't we use the ErrorMessage component for that?

Reply:
A situation where no dividends data is available is not an error, but a normal situation. The `ErrorMessage` component is intended for errors that can be fixed by the user. This is not the case here, so I'd prefer a short info text like this.